packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -372,3 +396,44 @@ class VideoPlayerOptions {
   /// currently no way to implement this feature in this platform).
   final bool mixWithOthers;
 }
+
+/// Subtitle option which embedded into video.
+class EmbeddedSubtitle {
+  /// Subtitle option which embedded into video.
+  ///
+  /// * It's recommended to not create direct instance by this constructor. Try to
+  ///   get from video player controller.
+  const EmbeddedSubtitle({
+    required this.language,
+    required this.label,
+    required this.trackIndex,
+    required this.groupIndex,
+    required this.renderIndex,
+  });
+
+  /// An instance of embedded subtitle which use to disable subtitle stream.
```

Comment:
An instance of the embedded subtitle that is used to disable the subtitle stream.